Deployment Failed

```
Run npm run deploy
> quietsato.github.io@1.0.0 deploy
> gatsby build && gh-pages -d public -b gh-pages
success compile gatsby files - 1.421s
success load gatsby config - 0.030s
success load plugins - 0.542s
success onPreInit - 0.004s
success initialize cache - 0.056s
success copy gatsby files - 0.098s
success Compiling Gatsby Functions - 0.249s
success onPreBootstrap - 0.258s
success createSchemaCustomization - 0.003s
success Checking for changed pages - 0.000s
success source and transform nodes - 0.091s
info Writing GraphQL type definitions to /home/runner/work/quietsato.github.io/quietsato.github.io/.cache/schema.gql
success building schema - 0.270s
success createPages - 0.000s
success createPagesStatefully - 0.087s
info Total nodes: 38, SitePage nodes: 3 (use --verbose for breakdown)
success Checking for changed pages - 0.000s
success onPreExtractQueries - 0.000s
success extract queries from components - 1.768s
success write out redirect data - 0.016s
success Build manifest and related icons - 0.051s
success onPostBootstrap - 0.053s
info bootstrap finished - 6.812s
success write out requires - 0.004s
success Building production JavaScript and CSS bundles - 30.190s
success Building HTML renderer - 16.242s
success Execute page configs - 0.023s
success Caching Webpack compilations - 0.001s
success run queries in workers - 0.012s - 3/3 241.03/s
success Merge worker state - 0.001s
success Rewriting compilation hashes - 0.000s
success Writing page-data.json files to public directory - 0.009s - 3/3 341.11/s
success Building static HTML for pages - 2.083s - 3/3 1.44/s
success onPostBuild - 0.020s
Pages
┌ src/pages/404.tsx
│ ├   /404/
│ └   /404.html
└ src/pages/index.tsx
  └   /
  ╭────────────────────────────────────────────────────────────────────╮
  │                                                                    │
  │     (SSG) Generated at build time                                  │
  │   D (DSG) Deferred static generation - page generated at runtime   │
  │   ∞ (SSR) Server-side renders at runtime (uses getServerData)      │
  │   λ (Function) Gatsby function                                     │
  │                                                                    │
  ╰────────────────────────────────────────────────────────────────────╯
info Done building in 56.16768718 sec
Cloning into '/home/runner/work/quietsato.github.io/quietsato.github.io/node_modules/.cache/gh-pages/https!github.com!quietsato!quietsato.github.io'...
fatal: could not read Username for 'https://github.com/': No such device or address
Error: Process completed with exit code 1.
```

--- a/.github/workflows/deploy.yml
+++ b/.github/workflows/deploy.yml
@@ -17,4 +17,11 @@ jobs:
           node-version: 16
           cache: "npm"
       - run: npm install
-      - run: npm run deploy
+      - name: Build
+        run: npm run build
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./public
+          publish_branch: gh-pages

